Sources/Momento/messages/responses/cache/data/scalar/Delete.swift

```diff
@@ -0,0 +1,8 @@
+public enum DeleteResponse {
```

Comment:
is this delete different from the delete cache response? is there an API that deletes scalar values?